docs/source/languagesections/languagesections.js

```diff
@@ -52,24 +52,27 @@ $(function() {
   $('div.code-section ul.section-selector li,div.plain-section ul.section-selector li').click(function(evt) {
     evt.preventDefault();
 
-    var sel_class = $(this).attr('class').replace(' docutils container', '').replace('notranslate', '');
+    var sel_class = $(this).attr('class')
+      .replace(' docutils container', '')
+      .replace('notranslate', '')
+      .replace(' selected', '');
```

Comment:
Clicking the same cell currently sees `sel_class = "r selected"`. This gets rid of the selected.